plugins/domains/scribd.com/scribd.com-error.js

```diff
@@ -1,14 +1,48 @@
+const URL = require("url");
+
 module.exports = {
 
     re: require('./scribd.com').re,
 
-    getData: function(oembedError, options, cb) {
-        if (oembedError === 401){
-            cb(null, {
-                message: "Scribd doesn't support embedding of private documents"
-            })
+    mixins: [ "*" ],
+
+    provides: ["scribdData"],
+
+    getData: function(oembedError, cb, url, options, urlMatch, request) {
+
+        if (oembedError === 401 && /secret_password=/.test(url) && urlMatch.length === 3) {
+
+            var secret = URL.parse(url, true).query["secret_password"];
```

Comment:
Since you do not check anything here for defined, it is better to add `(\?|&)` into the regex above:

`/(?:\?|$)`secret_password=/.test(url)`